.github/workflows/ci.yml

```diff
@@ -130,6 +126,7 @@ jobs:
               data: fs.readFileSync(path)
             });
       - name: Upload LICENSE
+        continue-on-error: true
```

Comment:
This can error if the LICENSE is already there, currently ignoring this, we'll consider other methods later on